pkgs/tools/networking/networkmanager/openvpn/default.nix

```diff
@@ -20,7 +20,7 @@ in stdenv.mkDerivation {
   ];
 
   buildInputs = [ openvpn networkmanager ]
-    ++ lib.optionals withGnome [ gtk3 libsecret libnma ];
+    ++ (if withGnome then [ gtk3 libsecret libnma ] else [ glib ]);
```

Comment:
Let's just add `glib` unconditionally if it's always needed (it won't clash with one propagated from `gtk3`).